src/corona_hakab_model/affinity_matrix.py

```diff
@@ -33,67 +33,42 @@ def __init__(self, agents: Iterable[Agent], consts: Consts):
         self.logger = logging.getLogger("simulation")
 
         self.logger.info("Building new AffinityMatrix")
+
         self.circular_matrix_types = {}
         for i, cm in enumerate(self.consts.circular_matrices):
             self.circular_matrix_types[cm.name] = (i, cm)
+
         self.non_circular_matrix_types = {}
         for j, ncm in enumerate(self.consts.non_circular_matrices, len(self.circular_matrix_types)):
             self.non_circular_matrix_types[ncm.name] = (j, ncm)
-        self.depth = j + 1
+
+        self.clustered_matrix_types = {}
+        for l, clm in enumerate(self.consts.clustered_matrices,
```

Comment:
PEP 8: no variables named 'l'

Reply:
it was the same way before, just with j... it works...
if you prefer it this way, i declared l before the loop

Reply:
no no, by PEP 8 conventions, you shouldn't name a variable "l"

https://www.python.org/dev/peps/pep-0008/#names-to-avoid

Reply:
oh ok. makes more seance.
renamed to k.